src/Entity/Cover.php

```diff
@@ -114,6 +112,15 @@ class Cover
     private $filePath;
 
     /**
+     * @ApiProperty(
+     *     attributes={
+     *         "openapi_context"={
+     *             "type"="integer",
+     *             "example"=769822
```

Comment:
The format in the database in `2020-05-18 10:56:48` but is it an timestamp here?

Reply:
It is the `$size` ;-)

Reply:
These diffs not always easy to read :-)